Updating ITP overview description (#1004)
CodeYourFuture/curriculum: org-cyf-itp/content/overview/_index.md

The objective of this course is to give people with no prior software development experience an introduction to programming and thinking about data. Common next-steps after this course are to get an entry level job in tech or IT, or to continue learning more advanced software engineering techniques to prepare for a software engineering job.

> Primarily this is achieved by teaching programming concepts with JavaScript and Python and data concepts with SQL in spreadsheets. In addition, there is a focus on professional development skills such as communicating effectively and working well on a team.

Reviewer: ```suggestion
Primarily this is achieved by teaching programming concepts with JavaScript and Python and data concepts with SQL in spreadsheets. We also focus on professional development skills such as communicating effectively and working well on a team.
```
